src/aequitas_webapp/basic_upload.py

```diff
@@ -115,5 +115,6 @@ def uploaded_file(filetype):
         # os.remove('tmp.csv')
         return render_template('report.html', content=content)
 
+
 if __name__ == "__main__":
-    app.run()
+    application.run()
```

Comment:
This module has in a sense been doing double-duty: declaring the application, registering request handlers and, upon direct execution, running the app in development mode. (Is that three duties?)

In production, behind an nginx server, this module is not run as `__main__` as it would under direct execution or loaded through `python -m`. Rather, some importable module must provide an object that meets the WSGI standard, and which object is named `application`. And, now, that module is this one. (Four duties? That's Flask.)